style.css

```diff
@@ -372,3 +394,195 @@ footer p {
   position: absolute;
   left: 85%;
 }
+
+#contact-img2 #contact-img3 #contact-img4 {
+  display: none;
+}
+
+@media only screen and (min-width: 1080px) {
+  .background-image {
```

Comment:
- [ ] At the specified width, the navbar and the `intro `heading overlap as you can see below 👇. Please try to fix it in order to make your app look better. I suggest using margins 😎 
![image](https://user-images.githubusercontent.com/46839548/160248673-7466a330-3b5c-480b-8b32-92349c0b4b29.png)

Reply:
- [ ] Kindly make sure to align the text in the centre as per the Figma design.
- Your design:
<img width="956" alt="Screenshot 2022-03-26 at 10 01 51 PM" src="https://user-images.githubusercontent.com/70269507/160253674-adbc05fb-c26a-414c-83b2-a18155479a1f.png">

- Figma design:
<img width="624" alt="Screenshot 2022-03-26 at 10 01 27 PM" src="https://user-images.githubusercontent.com/70269507/160253679-c56266e2-855b-4054-88dd-2f7d22525ac2.png">